app/players/elves/druid.py

```diff
@@ -0,0 +1,17 @@
+from app.players.elves.elf import Elf
+
+
+class Druid(Elf):
+    def __init__(self, nickname: str,
+                 musical_instrument: str, favourite_spell: str) -> None:
```

Comment:
Fix indentation here and in all other similar cases move every parameter to the new line to increase the readability of your code